content/posts/newsletter/2020-03-15.md

```diff
@@ -109,17 +124,41 @@ How to setup Conversations - [på svenska](https://hund.linuxkompis.se/2020/02/1
 [JabberPL.org turns 15](https://twitter.com/jabberpl/status/1231030107273355265) - Congratulations! For their users they also [improved their service](https://twitter.com/jabberpl/status/1231032793968279553). 
 
 ## GSoC
-[Google Summer of Code 2020 has been announced](https://summerofcode.withgoogle.com/organizations/5474262808264704/), some XMPP projects have already shown interest to mentor students.
+[Google Summer of Code 2020 has been announced](https://summerofcode.withgoogle.com/organizations/5474262808264704/), some XMPP projects have already shown interest to mentor students. Remind the deadline was March 31, 2020 at 20:00 o'clock.
 
 ## Extensions and specifications
 
+With respect to the last month we mention what happened in February as well to ensure completeness. So due to the late publishing this is is likely already over yet. Therefore, please remind the specified dates for Last Call and Call for Experience.
+
```

Comment:
Again, I don't think we need to be unnecessarily apologetic about combining two months into one newsletter.

We should rather just not write about certain things that have expired as if they're still ongoing.